creating-virtual-machines/virtualized-hardware-configuration.adoc

```diff
@@ -161,12 +161,67 @@ spec:
 You can check list of available models
 https://github.com/libvirt/libvirt/blob/master/src/cpu_map/index.xml[here].
 
+Enabling cpu compatibility enforcement
```

Comment:
CPUNodeDiscovery (as other features now) can be also added dynamically.  meaning you can `kubectl edit kubevirt-config`  the ConfigMap and the changes will apply  (double check me on the command syntax).

Reply:
but this is not article about updating config map. It is about new functionality.